pyproject.toml

```diff
@@ -76,7 +76,7 @@ target-version = "py39"
 select = ["B", "E", "F", "I", "N", "S", "W", "PL", "PT", "UP", "RUF"]
 # Ignore import sorting for now until lines_after_imports is respected
 # by ruff and we can replace isort
-ignore = ["D203", "D212", "PLR0913", "PLR2004"]
+ignore = ["D203", "D212", "PLR0913", "PLR2004", "S"]
```

Comment:
Can you actually just remove this from the select column above?

Reply:
removed it from select and also removed from per-file-ignores which doesn't seem needed anymore.